csatar-plugins/csatar/models/AccidentLogRecord.php

```diff
@@ -74,6 +76,13 @@ class AccidentLogRecord extends Model
         ],
     ];
 
+    public $morphMany = [
+        'history' => [
+            \Csatar\Csatar\Models\History::class,
+            'name' => 'history',
+        ],
+    ];
+
```

Comment:
Ez nem lehetett volna része a trait-nek?

Reply:
Gondolkodtam rajta, de szerintem nem. Ez minden mondellen lehet egyéb, pl. a csapat estében, már egy van egy 'history' mező, ezért ott más a kapcsolat neve it: ```'change_history' => [
            \Csatar\Csatar\Models\History::class,
            'name' => 'history'
        ],```

Reply:
Felül lehet írni ezekben a spéci esetekben, nem?